lib/messagebird/client.rb

```diff
@@ -235,6 +237,25 @@ def voice_message_create(recipients, body, params={})
         params.merge({ :recipients => recipients, :body => body.to_s })))
     end
 
+    def call_create(source, destination, call_flow, params={})
```

Comment:
It seems this expects `call_flow` to be a JSON payload already, but I think it makes sense to make it an object: the idea of the SDK is to abstract implementation details like the underlaying transport away.

Reply:
good idea! implemented.